modules/web/js/ballerina/ast/node.js

```diff
@@ -271,6 +271,21 @@ class ASTNode extends EventChannel {
     }
 
     /**
+     * Filter matching children belonging to current scope from the predicate function
+     * @param predicateFunction a function returning a boolean to match filter condition from children
+     * @return {[ASTNode]} array of matching AST nodes
```

Comment:
Should be as follows: 

@param {func} a function returning a boolean to match filter condition from children.
@return {ASTNode[]} array of matching AST nodes.